docs/topics/wasm/wasm-js-interop.md

```diff
@@ -380,7 +380,7 @@ Although Kotlin/Wasm interoperability shares similarities with Kotlin/JS interop
 | **Long**                | Type corresponds to JavaScript `BigInt`.                                                                                                                                                                            | Visible as a custom class in JavaScript.                                                                                                            |
 | **Arrays**              | Not supported in interop directly yet. You can use the new `JsArray` type instead.                                                                                                                                  | Implemented as JavaScript arrays.                                                                                                                   |
 | **Other types**         | Requires `JsReference<>` to pass Kotlin objects to JavaScript.                                                                                                                                                      | Allows the use of non-external Kotlin class types in external declarations.                                                                         |
-| **Exception handling**  | Can't catch JavaScript exceptions.                                                                                                                                                                                  | Can catch JavaScript `Error` via the `Throwable` type. It can catch any JavaScript exception using the `dynamic` type.                                      |
+| **Exception handling**  | Starting from 2.0.0, can catch any JavaScript exception via `JsException` and `Throwable`                               | Can catch JavaScript `Error` via the `Throwable` type. It can catch any JavaScript exception using the `dynamic` type.                                      |
```

Comment:
```suggestion
| **Exception handling**  | Starting from Kotlin 2.0.0, it can catch any JavaScript exception via the `JsException` and `Throwable` types.                               | Can catch JavaScript `Error` via the `Throwable` type. It can catch any JavaScript exception using the `dynamic` type.                                      |
```